### PR TITLE
Yro/EDUCATOR 2416

### PR DESCRIPTION
**Pull request by yro**
### Description
Attempts to fix enqueueing issue and fix duplicate studio imports.

Related to:
https://github.com/edx/edx-video-worker/pull/19

### Testing
- [ ] Unit tests

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review:

FYI: @edx/educator-neem

### Post-review
- [ ] Rebase and squash commits

### CODE REVIEW

File: control/veda_heal.py [hunk 1]
```diff
@@ -62,36 +62,44 @@ def send_encodes(self):
...
-                # the Video is old enough and escapes from the time-span that HEAL is picking up on.
-                # cc Greg Martin
-                VAC = VALAPICall(
+            # Update to VAL is also happening for those videos which are already marked complete,
+            # All these retries are for the data-parity between VAL and VEDA, as calls to VAL api are
+            # unreliable and times out. For a completed Video, VEDA heal will keep doing this unless
+            # the Video is old enough and escapes from the time-span that HEAL is picking up on.
+            # cc Greg Martin
+            if not self.val_status and len(encode_list) > 0:
+                self.val_status = 'transcode_queue'
```
Comment (Qubad786) [inline control/veda_heal.py]: `self.val_status` in this clause is not used in rest of the flow. Can we remove this clause?

**Comment by Qubad786** [comment]
@muhammad-ammar may you also take a look?

File: control/control_env.py [hunk 2]
```diff
@@ -57,3 +57,6 @@
 NODE_COLORS_GREEN = '\033[92m'
 NODE_COLORS_RED = '\033[91m'
 NODE_COLORS_END = '\033[0m'
+
+HEAL_START = 6
+HEAL_END = 144
```
Comment (muhammad-ammar) [inline control/control_env.py]: @yro It would be great if we can add a comment about why we choose these particular values?

**Comment by muhammad-ammar** [comment]
@yro I have kickstarted a [sandbox](https://tools-edx-jenkins.edx.org/job/Sandboxes/job/CreateSandbox/9373/console) with your branch if you need testing.

**Pull request merged**